### PR TITLE
Added z-index for wvr drop down menu

### LINKED ISSUE #133: Fix Z-Index on dropdown overlaps
Currently a dropdown menu will always appear beneath a dropdown button, even when the button belongs to another dropdown component.

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #133

### CODE REVIEW

File: projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss [hunk 1]
```diff
@@ -2,27 +2,27 @@
...
-  
+
 }
 
 .wvr-dropdown {
 
   cursor: pointer;
   font-family: var(--wvr-font-family-sans-serif);
   position: relative;
-  width: min-content;
```
Comment (jeremythuff) [inline projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss:25]: This should be `var(--wvr-dropdown-menu-width)`

File: projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss [hunk 2]
```diff
@@ -43,18 +43,20 @@
 
   .open {
     animation: openDD var(--wvr-dropdown-menu-animation-speed) ease-in-out;
+    z-index: 1200;
+    background: var(--wvr-white);
```
Comment (jeremythuff) [inline projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss:47]: This should be `var(--wvr-dropdown-menu-background)`

File: projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss (hunk 1 above)
Comment (rmathew1011) [reply projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss:25]: width value changed to var(--wvr-dropdown-menu-width)

File: projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss (hunk 2 above)
Comment (rmathew1011) [reply projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss:47]: dropdown background color changed to var(--wvr-dropdown-menu-background)

**Pull request merged**